website/server/models/challenge.js

```diff
@@ -235,7 +235,18 @@ async function _addTaskFn (challenge, tasks, memberId) {
   });
 
   // Update the user
-  toSave.unshift(User.update({ _id: memberId }, updateTasksOrderQ).exec());
+  const addToChallengeTagSet = {
```

Comment:
can we add a comment explaining what this new code does? Also the `// Update the user` comment should be probably removed or moved right before `toSave.unshift..`